### PR TITLE
[feature] Add Poetry as dependency management system and packaging

### LINKED ISSUE #5: [feature] Add Poetry as dependency management system and packaging
Currently, we use the classical `setuptools` with a `setup.py` for building the package. However, with the changes introduced by [PEP517](https://www.python.org/dev/peps/pep-0517/), we can now use more modern and cleaner tools, like [Poetry](https://python-poetry.org/).

These are the issues that need to be resolved:

  * [ ]  Create of the `pyproject.toml` with the necessary information about dependencies
  * [ ]  Document changes into the `README.md` to show the package building process

### PR DESCRIPTION
**Pull request by zealberth**
This pull request closes #5. The following changes were made:

- Addition of poetry as a packaging and dependency management tool
- Reorganization of files

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+keywords = ["machine-learning", "data-science", "fuzzy-c-means", "clustering"]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+include = [
+    "LICENSE",
+]
+
+[tool.poetry.dependencies]
```
Comment (omadson) [inline pyproject.toml:20]: I think the only dependence on fuzzy c-means is numpy. You do not need to add other libraries.

Put them as development dependencies.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "fcmeans"
+version = "0.1.0"
```
Comment (omadson) [inline pyproject.toml:3]: fuzzy-c-means is on 1.0.0 release on pypi. can you change this?

**Review by omadson — CHANGES_REQUESTED** [review]
Make the required changes and send a push.

**Comment by omadson** [comment]
just one more thing:
  - add your name to the `README.md` contributors section.

**Comment by zealberth** [comment]
@omadson could you verify if is everything ok now? thanks

**Review by omadson — APPROVED** [review]

**Pull request merged**